src/provider/match.js

```diff
@@ -149,6 +170,33 @@ async function check(url) {
 		logger.debug(e, 'Failed to decode and extract the bitrate');
 	}
 
+	if (!song.br) {
+		if (isHost('qq.com') && song.url.includes('.m4a')) {
+			//m4a is the lowest audio quality of qq music, usually 96kbps
+			song.br = 96000;
+		}
+
+		if (isHost('bilivideo.com') && song.url.includes('.m4a')) {
+			const re = /-(\d+)k/;
+			const result = song.url.match(re);
```

Comment:
```suggestion
			const result = song.url.match(/-(\d+)k\.m4a/);
```

应该没有单独增加 re 变量的必要。
另外推荐加入后缀，以防其他参数出现类似的字符。